Tools/clinic/clinic.py

```diff
@@ -4246,7 +4238,7 @@ def directive_version(self, required):
 
     def directive_module(self, name):
         fields = name.split('.')
-        new = fields.pop()
+        _ = fields.pop()
```

Comment:
I'd personally find this more idiomatic:

```py
fields - name.split[:-1]
```

But this is also fine :)

Reply:
I agree! I was just trying to keep the diff down, but let's exploit the moment and make this nicer :) Feel free to update the PR and land it; I'm on mobile for the rest of the day.